pkg/networkservice/chains/forwarder/server.go

```diff
@@ -79,6 +79,7 @@ func NewServer(ctx context.Context, name string, authzServer networkservice.Netw
 	)
 	nsClient := registryclient.NewNetworkServiceRegistryClient(ctx, clientURL, registryclient.WithDialOptions(clientDialOptions...))
 
+	netNsInfo := memif.NewNetNSInfo()
```

Comment:
@Mixaster995  Why are we doing this outside the memif.NewServer() ?  If this can be safely done inside memif.NewServer it vastly simplifies the use of memif.NewServer...

Reply:
@edwarnicke i was under impression that it was required for some logic, but was mistaken. Made some refactoring and tested all changes.

Reply:
@Mixaster995 Its all good.  When reworking someone else's work, its sometimes hard to figure out if something that looks weird is essential or not.